cedar-wasm/CHANGELOG.md

```diff
@@ -16,3 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Exposed types through `tsify` for `ValidateCall` and the schema. (#692)
 - Exposed cedar-wasm functionality for formatter and schema, context, and entity parsing: `wasm_format_policies`, `check_parse_schema`, `check_parse_context`, `check_parse_entities`. (#718)
 - Exposed cedar-wasm functionality for template parsing: `check_parse_template`.
+
+### Changed
+
+- Update `policyTextToJson` so that it reliably errors for all policies that
```

Comment:
Are these quoted names functions?

Reply:
They're the JS binding function names from annotations like `#[wasm_bindgen(js_name = "checkParseTemplate")]`

Reply:
Got it. Thanks.